src/db.ts

```diff
@@ -231,6 +231,22 @@ export const getPerson = async (personId: string): Promise<MockPerson> => {
   return augmentPerson(person);
 };
 
+export const removePerson = async (personId: string): Promise<MockPerson> => {
```

Comment:
would it have sense to delete also other dependant keys? mobile numbers, taxIdentifications, devices, challenges, etc?

Reply:
nice catch fixed within this commit [896bd11](https://github.com/kontist/mock-solaris/pull/169/commits/896bd116ff0a8b5e83d27f4615c126b88310154f) and regarding the challange i found that it's already deleted after each verification so there is no need to delete it again here 🙏